src-gui/Makefile

```diff
@@ -28,8 +28,12 @@ target/${APP}.jar: .deps/node .deps/resources
 clean:
 	rm -f .deps/node
 	rm -f .deps/resources
+	rm -f src/main/resources/ui/index.html
```

Comment:
No need to remove just add it in gitignore

Reply:
Done.